aries-site/src/layouts/content/Example/Container.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, defaultProps } from 'grommet';
+
+// This creates the overall canvas/frame that the Example sits inside of
+export const Container = ({
+  designer,
+  docs,
+  figma,
+  height: heightProp,
+  horizontalLayout,
+  plain,
+  screenContainer,
+  showResponsiveControls,
+  template,
```

Comment:
With the absence of docs, I wonder if we could add comments here (or wherever makes sense) which briefly explains what each of these props are/do? Thoughts?

Reply:
Maybe this makes more sense on Example.js?

Reply:
I'll add the comments on `Example.js` then reference that here.